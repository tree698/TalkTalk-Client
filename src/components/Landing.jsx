import React from 'react';
import { Outlet } from 'react-router-dom';
import Heading from './Heading';

export default function Landing() {
  return (
    <>
      <Heading />
      <Outlet />
    </>
  );
}
