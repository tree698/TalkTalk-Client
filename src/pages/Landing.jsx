import React from 'react';
import { Outlet } from 'react-router-dom';
import Heading from '../components/Heading';

export default function Landing() {
  return (
    <section className="w-full flex flex-col md:flex-row md:h-screen">
      <Heading />
      <Outlet />
    </section>
  );
}
