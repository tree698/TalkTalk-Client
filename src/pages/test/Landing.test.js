import renderer from 'react-test-renderer';
import Landing from '../Landing';

jest.mock('../../components/Heading');
jest.mock('../../components/Footer');

describe('Home', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Landing />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
