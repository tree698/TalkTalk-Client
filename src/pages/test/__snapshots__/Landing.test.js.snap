// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<section
  className="w-full h-screen flex flex-col md:flex-row"
>
  <div
    className="md:fixed md:bottom-1 md:left-1 text-[10px] md:text-xs lg:text-sm"
  />
</section>
`;
